index.js

```diff
@@ -37,11 +37,25 @@ server.route({
       reply(Boom.badRequest(validated.error.message));
     } else {
       var instance = validated.value;
-      reply(instance).code(201);
+      reply().code(201).header("Location: /v1/instances/5");
     }
   }
 });
 
+server.route({
+	method: 'GET'
+,	path: '/v1/instances/5'
```

Comment:
This route will only work for a specific instance, with id of `5`. Would we want to make this a parameter instead? I'd suggest reading the docs for Hapijs to get familiar with routing

http://hapijs.com/tutorials/routing